apps/dashboard/app/models/batch_connect/app.rb

```diff
@@ -106,31 +104,33 @@ def description
     # Default description for the batch connect app
     # @return [String] default description of app
     def default_description
-      ood_app.manifest.description
+      OodApp.instance_method(:manifest).bind(self).call.description
```

Comment:
There may be a better way to do this. Calling `super` calls the method of the same name - i.e., `default_description`. Only `default_description` doesn't exist on the parent, so we have to do some fancy footwork here.